### DISCUSSION TITLE
Choosing the location of the next expansion.

### DISCUSSION THREAD

**Discussion by VZozul** [sunflower-land/sunflower-land#2582] [Ideas]
### This is what the game looks like now, everything is very convenient, everything is visible.

![qw](https://github.com/sunflower-land/sunflower-land/assets/103659539/8b258e97-1e2b-48b6-944a-2ef8b81c434c)

### **But if the next expansions go down, then this is how it will look ..** 

![qw2](https://github.com/sunflower-land/sunflower-land/assets/103659539/bcdd4d47-b4b5-47d4-ab81-a01fafe94988)

**Maybe it’s convenient for someone, but I would like to have a choice of where to expand the island, now I have (first screen) another place 3 on the left and 3 on the right. Why not make a choice? So the islands would differ slightly from each other in forms, this is also interesting.**

**Comment by Chucrutty** [comment]
we need a calendar with the dates when the new expansions will be released
